vortex-ipc/src/reader.rs

```diff
@@ -336,6 +342,17 @@ impl<R: Read> StreamMessageReader<R> {
         Some(unsafe { root_unchecked::<Message>(&self.message) })
     }
 
+    pub fn next_raw(&mut self, read: &mut R) -> VortexResult<Buffer> {
```

Comment:
A larger refactor of the IPC reader can remove this weirdness along with lending iterators.